.env

```diff
@@ -0,0 +1,5 @@
+BROWSER=none
+EXTEND_ESLINT=true
+FAST_REFRESH=true
+
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Should be a new line in the end of this file https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline